<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-PMTCCV5S" failures="2" tests="4" name="SinTest" time="0.028" errors="0" timestamp="2022-01-25T13:15:22 IST">
  <testcase classname="TrignometricTests.SinTest" name="sinnegativeTest" time="0.002"/>
  <testcase classname="TrignometricTests.SinTest" name="sinnegativeTest" time="0.002">
    <failure type="java.lang.AssertionError" message="Method &amp;apos;sinTest&amp;apos; did not return expected value expected [0.0] but found [-2.4492935982947064E-16]">
      <![CDATA[java.lang.AssertionError: Method 'sinTest' did not return expected value expected [0.0] but found [-2.4492935982947064E-16]
at TrignometricTests.SinTest.sinnegativeTest(SinTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- sinnegativeTest -->
  <testcase classname="TrignometricTests.SinTest" name="sinpositiveTest" time="0.002">
    <failure type="java.lang.AssertionError" message="Method &amp;apos;sinTest&amp;apos; did not return expected value expected [0.0] but found [1.2246467991473532E-16]">
      <![CDATA[java.lang.AssertionError: Method 'sinTest' did not return expected value expected [0.0] but found [1.2246467991473532E-16]
at TrignometricTests.SinTest.sinpositiveTest(SinTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- sinpositiveTest -->
  <testcase classname="TrignometricTests.SinTest" name="sinpositiveTest" time="0.002"/>
</testsuite> <!-- SinTest -->
